modules/installation-azure-config-yaml.adoc

```diff
@@ -100,18 +124,25 @@ ifdef::gov[]
     region: usgovvirginia
 endif::gov[]
     resourceGroupName: existing_resource_group <9>
+    networkResourceGroupName: vnet_resource_group <10>
+    virtualNetwork: vnet <11>
+    controlPlaneSubnet: control_plane_subnet <12>
+    computeSubnet: compute_subnet <13>
```

Comment:
There are no ifdef and endif for these lines related to `azure-restricted`. These lines will be included in all the other azure documentations.